### PR TITLE
[MM-12454] fix formated message when have *** and improve the copy text button

### PR DESCRIPTION
**Pull request by cpanato**
#### Summary
the  `FormattedMarkdownMessage` was parsing the FAKE secret text so I update to use `FormattedMessage` when need to display the fake secret

Also, improve the copy button to show only when you click to show the secret.

#### Ticket Link
Jira: https://mattermost.atlassian.net/browse/MM-12454

#### Checklist
- [x] Ran `make check-style` to check for style errors (required for all pull requests)
- [x] Ran `make test` to ensure unit and component tests passed
- [x] Added or updated unit tests (required for all new features)

### CODE REVIEW

File: components/integrations/installed_oauth_app.jsx [hunk 1]
```diff
@@ -184,6 +194,22 @@ export default class InstalledOAuthApp extends React.PureComponent {
...
+            clientSecret = (
+                <span className='item-details__token'>
+                    <FormattedMarkdownMessage
+                        id='installed_integrations.client_secret'
+                        defaultMessage='Client Secret: **{clientSecret}**'
+                        values={{
+                            clientSecret: this.state.clientSecret,
+                        }}
+                    />
+                    <CopyText
```
Comment (lieut-data) [inline components/integrations/installed_oauth_app.jsx]: 0/5, but I think it makes sense to allow the copy without having to show the secret (useful for over screen snoopers).

**Comment by cpanato** [comment]
@jasonblais done, PTAL

File: components/integrations/installed_oauth_app.jsx (hunk 1 above)
Comment (cpanato) [reply components/integrations/installed_oauth_app.jsx]: imho if you copy the secret with the *** in place the user can feel this is not too secure....

**Comment by jasonblais** [comment]
Thanks @cpanato - looks like the build didn't pass.

**Comment by cpanato** [comment]
@jasonblais done, sorry about that

**Comment by cpanato** [comment]
cherry pick 1bd8a8e146d49d6e709aafd1486a0cd845896e21 to release 5.4 done

**Pull request merged**